secrets-provider-chart/templates/secrets-provider.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Values.global.namespace }}
+spec:
+  template:
+    spec:
+      serviceAccountName: {{ .Values.rbac.serviceAccount.name }}
+      containers:
+      - image: {{ .Values.secretsProvider.image }}:{{ .Values.secretsProvider.tag }}
+        imagePullPolicy: Always
```

Comment:
replace 'Always' to 'IfNotPresent'

Reply:
This will be delivered to the customer @oburstein-hub so I would imagine they wouldn't have the image locally no?

Reply:
This flag means that the image should be pulled to K8S from docker registry on every pod run (Always), or only when not pulled yet (IfNotExists)

Reply:
> only when not pulled yet

from first run?

Reply:
yes, from some previous run